### DISCUSSION TITLE
Project Framework and Data Sourcing - How to approach the goal of predicting NBA game outcomes?

### DISCUSSION THREAD

**Discussion by jeffjohannsen** [NBA-Betting/NBA_Betting#52] [Ideas]
**Project Framework**: How should we structure the approach to the problem? What is a good mental model for thinking about how to predict NBA game outcomes?

**Data Sourcing**: How do we choose the most impactful data types and the best sources for our predictive model? We value your input on pinpointing essential data and effective acquisition strategies.

[Current Framework](https://github.com/NBA-Betting/NBA_Betting#plan)
[Possible Future Framework](https://github.com/NBA-Betting/NBA_Betting#custom-model-architecture)

**Comment by Dinortchio** [comment]
Yo Jeff, 

Love the Project, didn't test it. 
When it comes to features and tune the ML Model, I beleive there is An Edge of Winners With some knowledge of sport. 
For Example, taking some narratifs in considerations, revenge mood, after Long road trip, before crucial matchup. 

We could also envisage some design end users and why not using Data Tools.
